src/sentry/static/sentry/app/views/settings/account/apiApplications/details.tsx

```diff
@@ -0,0 +1,100 @@
+import {RouteComponentProps} from 'react-router/lib/Router';
+import React from 'react';
+
+import {ApiApplication} from 'app/types';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {t} from 'app/locale';
+import AsyncView from 'app/views/asyncView';
+import ConfigStore from 'app/stores/configStore';
+import Form from 'app/views/settings/components/forms/form';
+import FormField from 'app/views/settings/components/forms/formField';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import TextCopyInput from 'app/views/settings/components/forms/textCopyInput';
+import apiApplication from 'app/data/forms/apiApplication';
+import getDynamicText from 'app/utils/getDynamicText';
+
+type State = {
+  app: ApiApplication;
+};
+
+class ApiApplicationsDetails extends AsyncView<
+  RouteComponentProps<{}, {}> & AsyncView['props'],
```

Comment:
@billyvg So this is how you should do route props. I guess I should remove `RouterProps` from our types file and use this instead.

Reply:
is `<{}, {}>` not the default?